### PR TITLE
Blog article on 'istioctl x describe'

### PR DESCRIPTION
**Pull request by esnible**
This is a proposed blog article introducing "istioctl experimental describe".  It is in tutorial format.

Resolves https://github.com/istio/istio.io/issues/4881

### CODE REVIEW

File: content/en/blog/2019/istioctl-describe/index.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
+---
+title: "'istioctl describe' in Istio 1.3"
+description: Introducing a command-line utility to find configuration affecting pod traffic.
+publishdate: 2019-09-20
```
Comment (rcaballeromx) [inline content/en/blog/2019/istioctl-describe/index.md]: @esnible: Why is this a release blocker if it won't be published until September 20? Please confirm that this is a blocker for Istio 1.3

File: content/en/blog/2019/istioctl-describe/index.md [hunk 2]
```diff
@@ -0,0 +1,244 @@
+---
+title: "'istioctl describe' in Istio 1.3"
+description: Introducing a command-line utility to find configuration affecting pod traffic.
```
Comment (rcaballeromx) [inline content/en/blog/2019/istioctl-describe/index.md]:
```suggestion
description: Introducing a CLI to discover the configuration affecting pod traffic.
```

File: content/en/blog/2019/istioctl-describe/index.md [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+publishdate: 2019-09-20
+attribution: Ed Snible (IBM)
+keywords: [traffic-management, istioctl, debugging]
+---
+
+Istio 1.3 includes an experimental command-line tool designed to help find and
```
Comment (rcaballeromx) [inline content/en/blog/2019/istioctl-describe/index.md]: Please be specific. This is not a new command line tool. This is a new subcommand for `istioctl` correct? If that's the case then please refer to it as such everywhere on the page. For example:
```suggestion
In Istio 1.3, we included the experimental `describe pod` subcommand for `istioctl`. We designed this tool to help find and
```

File: content/en/blog/2019/istioctl-describe/index.md [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+understand configuration that affects traffic to a pod.
```
Comment (rcaballeromx) [inline content/en/blog/2019/istioctl-describe/index.md]: There is nothing wrong with your use but `affect` and `effect` can be very confusing for non-English native readers. Consider:
```suggestion
understand the configuration that impacts the traffic to a pod.
```

File: content/en/blog/2019/istioctl-describe/index.md [hunk 5]
```diff
@@ -0,0 +1,244 @@
...
+
+`istioctl experimental describe pod <pod>` shows the Istio configuration that affects a pod.
```
Comment (rcaballeromx) [inline content/en/blog/2019/istioctl-describe/index.md]: This sentence is redundant. You already provided this information above.
```suggestion
```

File: content/en/blog/2019/istioctl-describe/index.md [hunk 6]
```diff
@@ -0,0 +1,244 @@
...
+
+The team has prepared a mini-tutorial to demonstrate the features and use of the tool.
+
+## describe tutorial using Bookinfo tutorial
```
Comment (rcaballeromx) [inline content/en/blog/2019/istioctl-describe/index.md]: This content should not be part of the blog post. Please create a task for this feature. See our [guide on page types](https://istio.io/about/contribute/creating-and-editing-pages/#choosing-a-page-type) for more details. 
@esnible I'll be happy to help you or any other member of the WG polish this content for publication.

File: content/en/blog/2019/istioctl-describe/index.md [hunk 7]
```diff
@@ -0,0 +1,244 @@
...
+{{< text plain >}}
+Pod: ratings-v1-f745cf57b-qrxl2
+   Pod Ports: 9080 (ratings), 15090 (istio-proxy)
+--------------------
+Service: ratings
+   Port: http 9080/HTTP
+Pilot reports that pod enforces HTTP/mTLS and clients speak HTTP
+{{< /text >}}
+
+## `DestinationRules`
```
Comment (rcaballeromx) [inline content/en/blog/2019/istioctl-describe/index.md]: Do not use the value of the `kind:` field unless you are referencing the value directly. See our style guide on [consistent capitalization](https://istio.io/about/contribute/style-guide/#use-consistent-capitalization)

File: content/en/blog/2019/istioctl-describe/index.md (hunk 1 above)
Comment (esnible) [reply content/en/blog/2019/istioctl-describe/index.md]: It is not a blocker.  We can publish it when it is ready.

**Comment by esnible** [comment]
Closing this blog PR in favor of doc PR https://github.com/istio/istio.io/pull/4971 .

**Pull request closed without merging**